src/js/components/Select/SelectContainer.js

```diff
@@ -460,7 +460,13 @@ const SelectContainer = forwardRef(
                         if (optionRef) optionRef.current = node;
                         if (optionActive) activeRef.current = node;
                       }}
-                      tabIndex={optionSelected ? '0' : '-1'}
+                      tabIndex={
```

Comment:
If I:
- open select
- tab to options
- use arrow keys to go down some options
- use shift tab

Then I would expect the shift tab to go back to search, but it goes back to the first option (even though it's not selected?)

https://deploy-preview-7675--sad-tereshkova-173d07.netlify.app/?path=/story/input-select-search--search&globals=theme:hpe

https://github.com/user-attachments/assets/080eeb0f-f2ea-4760-9968-aa6e6f98013d